src/GNDStk.hpp

```diff
@@ -1,10 +1,74 @@
-#ifndef GNDSTK_HPP
-#define GNDSTK_HPP
 
-#include "knoop.hpp"
+#pragma once
 
-namespace njoy{
-  namespace gndstk {
-  }
-}
-#endif // GNDSTK_HPP
+// ------------------------
+// External libraries
+// ------------------------
+
+#include "pugixml.hpp"
+#include "json.hpp"
+
+
+// ------------------------
+// C++
+// ------------------------
+
+// miscellaneous
+#include <cassert>
+#include <cctype>
+#include <fstream>
+#include <iomanip>
+#include <iostream>
+#include <tuple>
+#include <type_traits>
+#include <typeinfo>
+#include <variant>
+
+// sequence containers,
+// except array and
+// forward_list
+#include <deque>
+#include <list>
+#include <vector>
+
+// boost demangle,
+// if it's wanted
+#ifdef GNDS_BOOST
+#include <boost/core/demangle.hpp>
+#endif
+
+
+// ------------------------
+// GNDS
+// ------------------------
+
+namespace gnds {
```

Comment:
OK, I changed that a while back. I think there's a way to alias a namespace nowadays, so someone who wants an easily-typed lower-case namespace, which was my intention, can have it without much trouble. And there's always the good old-fashioned "using namespace."

Reply:
I want to avoid alias or `using` statements to get around this. If we had lots of legacy code, that would be a different story. Let's not introduce legacy code that we will have to support for many years right from the beginning.